server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/VariableWidthHistogramAggregationBuilder.java

```diff
@@ -46,21 +46,20 @@
 public class VariableWidthHistogramAggregationBuilder
     extends ValuesSourceAggregationBuilder<ValuesSource.Numeric, VariableWidthHistogramAggregationBuilder> {
 
+    private int numBuckets = 10;
```

Comment:
Sorry, could you keep the non-static stuff below the static stuff? I think I said the wrong thing when I left a message earlier. Sorry!